tutorials/cuda_mps/README.md

```diff
@@ -73,6 +73,11 @@ MPS, it can also improve the overall GPU utilization.
 
 ## Performance Benefits on x86 System 
 
+> **Note:** Endoscopy Tool Tracking does not work with CUDA MPS after [holohub/Holoscan-SDK-v2.6.0](https://github.com/nvidia-holoscan/holohub/tree/holoscan-sdk-2.6.0) due to the
+> unavailability of CUDA dynamic parallelism implemented in this
+> [PR](https://github.com/nvidia-holoscan/holohub/pull/550) in CUDA MPS. In case endoscopy tool tracking needs
+> to be tested with CUDA MPS, please use a HoloHub tag before the mentioned PR.
```

Comment:
Nitpick, could we explicitly state to use the tag? Slightly redundant with the first line, but I think this helps clarify that the breaking PR was merged after the tag:
```suggestion
> to be tested with CUDA MPS, please use the [`holoscan-sdk-v2.6.0`](https://github.com/nvidia-holoscan/holohub/tree/holoscan-sdk-2.6.0) tag or earlier.
```